cmd/awscollector/Dockerfile

```diff
@@ -11,6 +11,9 @@ FROM alpine:latest AS certs
 
 RUN apk --update add ca-certificates
 
+FROM alpine:latest AS ecstarget
+RUN touch ecs_sd_targets.yaml
```

Comment:
Can we just have a blank yaml file committed to the repo that can be copied in the copy stage?